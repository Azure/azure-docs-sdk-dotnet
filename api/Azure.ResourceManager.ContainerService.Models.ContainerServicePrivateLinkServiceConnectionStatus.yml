### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus&text=ContainerServicePrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.op_Equality(Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus,Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.op_Inequality(Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus,Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Equals(Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
commentId: T:Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
namespace: Azure.ResourceManager.ContainerService.Models
name: ContainerServicePrivateLinkServiceConnectionStatus
fullName: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
nameWithType: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.ContainerService.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ContainerService v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ContainerService v1.2.0-beta.2
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct ContainerServicePrivateLinkServiceConnectionStatus : IEquatable<Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type ContainerServicePrivateLinkServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure ContainerServicePrivateLinkServiceConnectionStatus

      Implements IEquatable(Of ContainerServicePrivateLinkServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private link service connection status.
metadata:
  api_name:
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.ContainerService.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
  - Azure::ResourceManager::ContainerService::Models::ContainerServicePrivateLinkServiceConnectionStatus
  - ContainerServicePrivateLinkServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
