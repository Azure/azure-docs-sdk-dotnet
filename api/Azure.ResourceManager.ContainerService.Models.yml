### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolAvailableVersion
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolAvailableVersions
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolNetworkPortRange
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolNetworkProfile
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolUpgradeProfilePropertiesUpgradesItem
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ArmContainerServiceModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceDateSpan
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceEndpointDependency
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceEndpointDetail
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetCredentialResult
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetCredentialResults
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetHubProfile
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetPatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceIPTag
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceLinuxProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceAbsoluteMonthlySchedule
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceRelativeMonthlySchedule
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceSchedule
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceWeeklySchedule
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceWindow
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfileForSnapshot
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfileKubeProxyConfig
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfileKubeProxyIPVSConfig
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOSOptionProperty
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOutboundEnvironmentEndpoint
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkResourceData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceSshConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceSshPublicKey
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTagsObject
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTimeInWeek
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTimeSpan
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTrustedAccessRole
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTrustedAccessRoleRule
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceUserAssignedIdentity
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.KubeletConfig
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.LinuxOSConfig
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAadProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAccessProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAddonProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAddonProfileIdentity
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAgentPoolProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAgentPoolProfileProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterApiServerAccessProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAutoScalerProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterAutoUpgradeProfile
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterCredential
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterCredentials
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterGuardrailsProfile
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterHttpProxyConfig
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterIngressProfileWebAppRouting
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterLoadBalancerProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterLoadBalancerProfileManagedOutboundIPs
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterMonitorProfileKubeStateMetrics
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterMonitorProfileMetrics
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterNatGatewayProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterOidcIssuerProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPodIdentity
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPodIdentityException
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPodIdentityProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPoolUpgradeProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPoolUpgradeProfileUpgradesItem
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPropertiesForSnapshot
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterRunCommandContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterRunCommandResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSecurityProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSecurityProfileAzureDefender
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSecurityProfileDefender
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSecurityProfileImageCleaner
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSecurityProfileKeyVaultKms
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterServicePrincipalProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterStorageProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterStorageProfileDiskCsiDriver
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterVerticalPodAutoscaler
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterWindowsProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterWorkloadAutoScalerProfile
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.SysctlConfig
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.WindowsGmsaProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolNetworkPortProtocol
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AgentPoolType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.AutoScaleExpander
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetMemberProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceFleetProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceLoadBalancerSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceMaintenanceRelativeMonthlyScheduleWeekIndex
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkPlugin
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkPluginMode
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkPolicy
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfileKubeProxyIPVSScheduler
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceNetworkProfileKubeProxyMode
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOSDiskType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOSSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOSType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceOutboundType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceStateCode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceTrustedAccessRoleBindingProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ContainerServiceWeekDay
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.EbpfDataplane
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.GpuInstanceProfile
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.IPFamily
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.KubeConfigFormat
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.KubeletDiskType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterGuardrailsProfileLevel
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterKeyVaultNetworkAccessType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterLoadBalancerBackendPoolType
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterNodeOSUpgradeChannel
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterNodeResourceGroupRestrictionLevel
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterPodIdentityProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSkuName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterSkuTier
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterVerticalPodAutoscalerUpdateMode
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ManagedClusterWorkloadAutoScalerControlledValue
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ScaleDownMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ScaleSetEvictionPolicy
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.ScaleSetPriority
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.SnapshotType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.UpgradeChannel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.WindowsVmLicenseType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerService.Models.WorkloadRuntime
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.ContainerService.Models
commentId: N:Azure.ResourceManager.ContainerService.Models
name: Azure.ResourceManager.ContainerService.Models
fullName: Azure.ResourceManager.ContainerService.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.ContainerService.Models
  - Azure::ResourceManager::ContainerService::Models
  - Models
  - ContainerService.Models
  - ContainerService::Models
  helpviewer_keywords:
  - Azure.ResourceManager.ContainerService.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
