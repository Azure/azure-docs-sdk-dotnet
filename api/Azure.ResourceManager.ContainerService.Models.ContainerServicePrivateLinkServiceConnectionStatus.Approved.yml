### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus&text=ContainerServicePrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  nameWithType: ContainerServicePrivateLinkServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As ContainerServicePrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.ContainerService.Models
name: Approved
fullName: Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
nameWithType: ContainerServicePrivateLinkServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.ContainerService.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ContainerService v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ContainerService v1.2.0-beta.2
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.ContainerService.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ContainerService.Models.ContainerServicePrivateLinkServiceConnectionStatus.Approved
  - Azure::ResourceManager::ContainerService::Models::ContainerServicePrivateLinkServiceConnectionStatus::Approved
  - Approved
  - ContainerServicePrivateLinkServiceConnectionStatus.Approved
  - ContainerServicePrivateLinkServiceConnectionStatus::Approved
  helpviewer_keywords:
  - ContainerServicePrivateLinkServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct ContainerServicePrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
